{"uuid":"13db9792-059c-4c2e-a157-3bb5bf637534","name":"Scenario: User enters Invalid Python code and Presented with Error Message","children":["80b73b03-31bf-40c8-8a1b-9a56241da50c"],"befores":[],"afters":[{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687875471624,"stop":1687875471624},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687875471867,"stop":1687875471867},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687875471924,"stop":1687875471924},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687875471968,"stop":1687875471969},{"name":"hooks.AppHook.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1687875471976,"stop":1687875471976}],"start":1687875471532,"stop":1687875471976}